__tests__/app.test.js

```diff
@@ -30,25 +42,65 @@ describe('GET/api/topics', () => {
       })     
 });
 
-describe('/GET incorrect path', () => {
-    test('Responds with 404 if incorrect path specified', () => {
+
+describe('/GET api', () => {
+    test('Responds with information on available api endpoints', () => {
         return request(app)
-        .get('/api/notTopics')
-        .expect(404)
+        .get('/api')
+        .expect(200)
         .then(({ body }) => {
-          expect(body.msg).toBe("not found")
+          expect(body).toEqual(endpoints)
         })  
       })
 });
 
 
-describe('/GET api', () => {
-    test('Responds information on available api endpoints', () => {
+describe('/GET api/articles:article_id', () => {
+
+    test('Responds with the correct article for the given article_id', () => {
+        let article = {
+            article_id: 1,
+            title: 'Living in the shadow of a great man',
+            topic: 'mitch',
+            author: 'butter_bridge',
+            body: 'I find this existence challenging',
+            created_at: '2020-07-09T20:11:00.000Z',
+            votes: 100,
+            article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+          }
         return request(app)
-        .get('/api')
+        .get('/api/articles/1')
         .expect(200)
-        .then(({ body }) => {
-          expect(body).toEqual(endpoints)
+        .then(({ body }) => {  
+        expect(body).toEqual(article)
```

Comment:
The assertions below are unnecessary as they were proven by the line above already